index.js

```diff
@@ -1,17 +1,54 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item); //adicionar um novo item
 
-  get(pos) {}
+    //esse array deve estar organizado em ordem crescente
+    this.items.sort((a, b) => a - b);
 
-  max() {}
+    this.length = this.items.length;
+    //manter o length atualizado
+  }
 
-  min() {}
+  get(pos) {
+    //garante que seja ordem crescente porque está chamando o add(item) que faz essa coisa
+    if (this.items[pos]) return this.items[pos];
+    throw new Error("OutOfBounds");
+  }
 
-  sum() {}
+  //retornar o valor mais alto da matriz
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return this.items[this.length - 1]; //-1 quer dizer que estou pegando sempre o último elemento do meu array que é sempre o maior
+  }
 
-  avg() {}
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    }
+    return this.items[0]; //quer o menor então vai ser sempre o primeiro item
+  }
+
+  sum() {
+    return this.items.reduce(
```

Comment:
Boa! Aqui daria pra ter feito o reduce em uma linha somente ;-)

return this.items.reduce((sum, num) => sum + num, 0);